README.md

```diff
@@ -76,17 +76,4 @@ The apps are now running in the port range of 8081 to 8090 on localhost.
 
 ### Development
 
-If you are a developer who has made changes to your local copy of OVE apps, and want to quickly rebuild it without rebuilding the docker container, you can run a container and mount the code as a volume:
-
-```sh
-cd /some/path/to/ove-apps
-docker run -it -p 8081-8090:8081-8090 -v $PWD:/code ove-apps bash
-```
-
-and then, inside the container, run:
-
-```sh
-cd /code
-lerna bootstrap --hoist && lerna run clean && lerna run build
-pm2 start pm2.json
-```
+If you are a developer who has made changes to your local copy of OVE apps, and want to quickly rebuild it with the **build.sh** script and run the app with the **docker-compose** file. The docker-compose.yml file can be customized to reflect local changes.
```

Comment:
"If you are a developer who has made changes to your local copy of OVE apps, ~~and want to~~ *you can* quickly rebuild it with the **build.sh** script and run the app with the **docker-compose** file."

Reply:
@jamesscottbrown Can you please suggest changes for single files instead of comments. It makes the whole process simpler.